client/src/pages/EditProfilePage/EditProfilePage.tsx

```diff
@@ -90,31 +190,155 @@ const EditProfilePage = () => {
               />
             </div>
           )}
-
-          <label className="block font-bold mb-2 text-sm" htmlFor="email">
-            Email
-            <input
-              className="bg-gray-800 p-2 rounded text-white w-full"
-              id="email"
-              name="email"
-              type="email"
-              value={formData.email}
-              onChange={handleChange}
-            />
+          <EditProfileInput
+            label="Email"
+            type="email"
+            name="email"
+            value={formData.email}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Nickname"
+            type="text"
+            name="nickName"
+            value={formData.nickName}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Cohort"
+            type="text"
+            name="cohort"
+            value={formData.cohort}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Personal Bio"
+            type="textarea"
+            name="personalBio"
+            value={formData.personalBio}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Current Company"
+            type="text"
+            name="currentPositionCompany"
+            value={formData.currentPositionCompany}
+            onChange={handleChange}
+          />
+          <EditProfileInput
+            label="Current Title"
+            type="text"
+            name="currentPositionTitle"
+            value={formData.currentPositionTitle}
+            onChange={handleChange}
+          />
+          <label className="block font-bold mb-2 text-sm" htmlFor="availabilityForNetworking">
+            Available for Networking?
+            <div
+              className={`cursor-pointer h-3 inline-block ml-2 mt-2 w-3 ${
+                formData.availabilityForNetworking ? 'bg-yellow-500' : 'bg-blue-500'
+              }`}
+              role="button"
+              tabIndex={0}
+              onClick={() =>
+                setFormData((prevData) => ({
+                  ...prevData,
+                  availabilityForNetworking: !prevData.availabilityForNetworking,
+                }))
+              }
+            ></div>
+            <span className="ml-2 text-xs">
+              {formData.availabilityForNetworking ? 'Yes' : 'No'}
+            </span>
           </label>
-          <label className="block font-bold mb-2 text-sm" htmlFor="personalBio">
-            Personal Bio
-            <input
-              className="bg-gray-800 p-2 rounded text-white w-full"
-              id="personalBio"
-              name="personalBio"
-              type="text"
-              value={formData.personalBio}
-              onChange={handleChange}
-            />
+          <EditProfileInput
```

Comment:
This is probably for a future ticket but it might be cool to make an array of objects out of these labels, types and names and map out all the EditProfileInput components from the array

Reply:
I agree! That will be a good ticket.